doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -305,7 +305,7 @@ Cellular samples (renamed from nRF9160 samples)
 
 * Added the :ref:`battery` sample to show how to use the :ref:`modem_battery_readme` library.
 
-* :ref:`nrf_cloud_mqtt_multi_service` sample:
+* :ref:`nrf_cloud_multi_service` sample:
```

Comment:
Changelog should mention name change and added functionality. I recommend adding ``  * Renamed Cellular: nRF Cloud MQTT multi-service to Cellular: nRF Cloud multi-service.`` as the very first entry in the list for the name change.